decidim-generators/lib/decidim/generators/app_generator.rb

```diff
@@ -120,6 +120,13 @@ def remove_sprockets_requirement
         gsub_file "config/environments/production.rb", /config\.assets.*$/, ""
       end
 
+      def patch_test_file
```

Comment:
I created this issue for actually fixing the workaround https://github.com/decidim/decidim/issues/14776